esphome/wizard.py

```
@@ -192,12 +201,14 @@ def wizard_write(path, **kwargs):
             kwargs[key] = sanitize_double_quotes(kwargs[key])
 
     if "platform" not in kwargs:
-        if board in esp8266_boards.ESP8266_BOARD_PINS:
+        if board in esp8266_boards.BOARDS:
             platform = "ESP8266"
-        elif board in rp2040_boards.RP2040_BOARD_PINS:
+        elif board in esp32_boards.BOARDS:
+            platform = "ESP32"
+        elif board in rp2040_boards.BOARDS:
             platform = "RP2040"
         else:
-            platform = "ESP32"
+            platform = "LIBRETINY"
```

This should actually be explicit for all cases and raise an error on else (unknown platform)